src/I18n/TranslatorRegistry.php

```diff
@@ -76,7 +76,7 @@ public function setCacher(CacheEngine $cacher)
      * @param string $name The translator package to retrieve.
      * @param string|null $locale The locale to use; if empty, uses the default
      * locale.
-     * @return \Aura\Intl\TranslatorInterface A translator object.
+     * @return \Aura\Intl\TranslatorInterface|null A translator object.
```

Comment:
There are a lot of `@return bla\bla\bla|null` and `@return bla\bla\bla|void` on functions that don't have a final return statement.

This also messes up inheritance, because I often remove the `null` from the `@return` and then PhpStorm complains that the signature is not the same as the derived class.

I think it's okay to say a method returns `@return StuffInterface|null` as long as all the return statements are there. Don't say it returns `null` and then do this.

```
    /**
     * @return Thing|void
     */
    function Foo($value) {
        if($value) {
            return new Thing();
        }
    }
```

I don't like the above, and think it should be done like this with the explicate null value. It clearly shows the programmers intent.

```
    /**
     * @return Thing|null
     */
    function Foo($value) {
        if($value) {
            return new Thing();
        }
        return null;
    }
```

On a side note:

It would be nice, and I'm just saying it would be nice. If methods that return an object type (i.c. class or interface) could not be mixed with a primitive type. As a coding standard. Therefore, things like `@return StuffInterface|bool` would be against the standard.